### ISSUE TITLE
R4i Upgrade Revolution for DS v1.4.1

### ISSUE THREAD

**Issue by nChuggeT** [kitling/flashcart_core#35]
ChipID: 00000FC2
HW Revision: 00000000

<details><summary>Images <em>(Click to Expand)</em></summary><p> 

![img_0210](https://user-images.githubusercontent.com/6129700/29741123-6f6b4996-8a5e-11e7-8eb8-0e824fe8b3b7.JPG)
![img_0211](https://user-images.githubusercontent.com/6129700/29741125-723ea082-8a5e-11e7-8515-18a5218ab2ab.JPG)
![img_0212](https://user-images.githubusercontent.com/6129700/29741126-73e4aa6c-8a5e-11e7-9163-4d91b4956779.JPG)

</p></details> 

GM9 Untrimmed Dump: 
[dump.zip](https://github.com/kitling/flashcart_core/files/1253605/dump.zip)

**Comment by kitling** [comment]
This is probably not a duplicate of #17 due to the different site name.

**Comment by ShadixAced** [comment]
@kitling Yes, the other is r4i-gold. This card is for DSi / DSi XL.  @nChuggeT Do you know which game the R4 spoofs ?

**Comment by nChuggeT** [comment]
@ShadixAced Yes, it's ALEX RIDER STORMBREAKER, but the firmware for the gold version did not work on this card, it said it was fake if I used that firmware.

**Comment by ShadixAced** [comment]
@nChuggeT Since it isn't a Gold card, it wouldn't be usable. This is an DSi/DSi XL flashcard. If you want to update the firm, just use it from r4i-sdhc.com

**Comment by Eddy119** [comment]
Hello, 
For your information, there are actually two carts which labels look identical.
http://www.r4wood.com/news/159/Problem-of-R4i-SDHC-V1.4.1-Update-for-DSi-1.4.2.html
I have the one without the chip at the back.
However, I have virtually no knowledge about computer jargon (yet), so I think this is all I can help with... Other than tell you guys that you're amazing :)

**Comment by kitling** [comment]
Could you try with the new release, v0.3.1? just added support for a bunch of carts from this site.

**Comment by nChuggeT** [comment]
Says its not supported, flashcart_core 7ac8

**Comment by Eddy119** [comment]
Mine has the flashchip ID 58454c41
I used one of the unstable versions on gbatemp to try dumping, but it only gave me FFFFFFF....
When I tried to restore, it bricked.
The current version, v0.3.1, says it's not supported.
I'll leave this here for someone to reverse engineer... www.r4i-sdhc.com/fwup142c.rar
Anyway, thank you for your work!

**Comment by nikhiljha** [comment]
https://i.imgur.com/PGO5ZeR.jpg

@kitling Same card, except the latest release (upon selecting R4i-SDHC family) flashes (apparently successfully) but then still continues to work to play DS games on my DSi and doesn't work for 3ds.guide.

Is there anything I can do to help get support for this kart?

**Comment by nikhiljha** [comment]
I went through all the options, the only one that worked was r4i-sdhc.

```
[INFO]: DSTT: Init
[NOTICE]: DSTT: Flashchip ID = 0x51e7c967
[INFO]: R4SDHC: Init
[ERROR]: R4SDHC: trySecureInit: status (13) not RAW and cannot reset
[ERROR]: R4SDHC: trySecureInit: status (13) not RAW and cannot reset
[ERROR]: R4SDHC: trySecureInit: status (13) not RAW and cannot reset
[ERROR]: R4SDHC: Secure init failed!
[INFO]: R4iGold: Init
[NOTICE]: R4iGold: HW Revision = 21df0811
[NOTICE]: R4iGold: HW Type = 472cf30d
[ERROR]: r4isdhc: found type 1 cart
[ERROR]: r4isdhc: found type 1 cart
```

I also made this file somehow...
[backup.bin.zip](https://github.com/ntrteam/flashcart_core/files/1598906/backup.bin.zip)

**Comment by HWNJ** [comment]
I have that card, if someone needs some info (dumps or anything) I can help!
Just request what you need!

**Comment by jPress05** [comment]
> I have that card, if someone needs some info (dumps or anything) I can help! Just request what you need!

@HWNJ Hey, would you happen to still have any complete dumps of this cards firmware that you would be happy to share?
